spateo/tools/cluster/find_clusters.py

```diff
@@ -233,5 +233,43 @@ def scc(
     )
 
     adata.obs[key_added] = clusters
+    adata.obs[key_added] = adata.obs[key_added].astype(str)
 
     return adata
+
+
+@SKM.check_adata_is_type(SKM.ADATA_UMI_TYPE)
+def optimize_cluster(adata: anndata.AnnData, radius: int = 50, key: str = "label") -> list:
+    """
+    Optimize the label by majority voting in the neighborhood.
+
+    Args:
+        adata: an Anndata object, after normalization.
+        radius: the radius of the neighborhood.
+        key: the key in `.obs` that corresponds to the cluster labels.
+    """
+    import ot
```

Comment:
are you using POT? POT has some dependency issues and we are planning to remove it. I think we can use SciPy instead of ot for distance calculations.

Reply:
I replaced the `POT` with `scipy` in the new Pull Request, although I lost some of the runtime efficiency.